src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -72,6 +73,27 @@
             .field(MapFetcher.field("agency_timezone"))
             .build();
 
+    // Represents rows from booking_rules.txt
+    public static final GraphQLObjectType bookingRulesType = newObject().name("booking_rules")
+            .description("A GTFS booking rule object")
+            .field(MapFetcher.field("id", GraphQLInt))
+            .field(MapFetcher.field("booking_rule_id"))
+            .field(MapFetcher.field("booking_type", GraphQLInt))
+            .field(MapFetcher.field("prior_notice_duration_min", GraphQLInt))
+            .field(MapFetcher.field("prior_notice_duration_max", GraphQLInt))
+            .field(MapFetcher.field("prior_notice_last_day"))
+            .field(MapFetcher.field("prior_notice_last_time"))
+            .field(MapFetcher.field("prior_notice_start_day"))
```

Comment:
Change to integer per https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#gtfs-bookingrules.